easybuild/easyconfigs/m/METIS/METIS-5.1.0-intel-2014b-32bitIDX.eb

```diff
@@ -0,0 +1,23 @@
+name = 'METIS'
+version = '5.1.0'
+versionsuffix = '-32bitIDX'
```

Comment:
Ah, that explains it.

Maybe add a comment right above the versionsuffix to clarify that? Something like "default 32-bit IDTYPEWIDTH, no patch used" ?